### DISCUSSION TITLE
Podman Desktop will not ignore self-signed certificates for registries

### DISCUSSION THREAD

**Discussion by kanedk** [podman-desktop/podman-desktop#6463] [General]
Hello all,

I'm following instructions to set up an insecure registry (using a self-signed cert)
https://podman-desktop.io/docs/containers/registries#setting-up-a-registry-with-an-insecure-certificate

/etc/hosts contains a valid entry for "docker.devhost" IP

I edit /etc/containers/registries.conf and add
```
[[registry]]
location = "docker.devhost"
insecure = true
```

then `pkill podman`

I can `podman pull docker.devhost/blah/someimage` just fine.

However, I don't see the registry listed in the Podman Desktop `Settings -> Registries` UI, and if I try to add it, it does not show me a window like
https://podman-desktop.io/assets/images/registry-warning-insecure-245d7254807b5939c082f47e5d538d80.png
to enable me to add the cert to a trust store.  Instead I get an error
`Unable to find auth info for https://docker.devhost/v2/. Error: RequestError: self signed certificate
`

Using podman on the command line has no problems working with this insecure registry, so what is going wrong with Docker Desktop?

Podman 4.9.3 / Podman Desktop 1.8.0 / Fedora 39

**Comment by taomanwai** [comment]
Similar issue here, and Podman Desktop shows "Unable to find auth info for https://artifactrepo.server.xxx:55743/v2/. Error: RequestError: Bad response: 504" instead. How to fix?
